lib/ansible/plugins/doc_fragments/default_callback.py

```diff
@@ -36,6 +36,16 @@ class ModuleDocFragment(object):
           - key: display_ok_hosts
             section: defaults
         version_added: '2.7'
+      display_includes:
+        name: Show dynamicly included tasks
+        description: "Toggle to control displaying dynamicly included task/host results in a task"
+        type: bool
+        default: yes
+        env:
+          - name: ANSIBLE_DISPLAY_INCLUDES
+        ini:
+          - key: display_includes
+            section: defaults
```

Comment:
```suggestion
      display_included_tasks:
        name: Show dynamically included task files
        description: "Toggle to control displaying dynamically included task files"
        type: bool
        default: yes
        env:
          - name: ANSIBLE_DISPLAY_INCLUDED_TASKS
        ini:
          - key: display_included_tasks
            section: defaults
```

Reply:
Since this is specific to task includes, not play includes, it should be named accordingly.